lib/model/folder_sendrecv.go

```diff
@@ -1757,6 +1765,11 @@ func (f *sendReceiveFolder) moveForConflict(name, lastModBy string, scanChan cha
 }
 
 func (f *sendReceiveFolder) newPullError(path string, err error) {
+	if errors.Cause(err) == f.ctx.Err() {
```

Comment:
I wonder if we don't do something like `fmt.Errorf("foo: %q", otherErr)`, which would kill Cause from working.
Did you check all paths for things that check the context?

Reply:
Good point, we did - now it's errors.Wrap everywhere.